### ISSUE TITLE
[Bug]:  Error when installing specified fastdup

### ISSUE THREAD

**Issue by zczczc11122** [visual-layer/fastdup#353]
### What happened?

`import os
os.environ['FASTDUP_PRODUCTION'] = '1'
import fastdup
fd = fastdup.create(work_dir='work_dir', input_dir='img_cache_folder')
fd.run(ccthreshold=0.97, d=960, overwrite=True, run_mode=1, high_accuracy=True)`

Running the above code reports an error
[FATAL] This fastdup version is deprecated, please reinstall from pypi using `pip install fastdup`

I tried reinstalling a higher version of fastdup，error reported again,
ERROR: Could not find a version that satisfies the requirement fastdup==2.6 (from versions: 1.85)
ERROR: No matching distribution found for fastdup==2.6

I tried to find the wheel file corresponding to version 2.5 or 2.6 from the link https://github.com/visual-layer/fastdup/releases, but could not find it. If I want to install a higher version of fastdup through wheel, where should I get the corresponding version of wheel?

### What did you expect to see?

A functioning fastdup environment

### What version of fastdup were you runnning on?

2.2

### What version of Python were you running on?

Python 3.8

### Operating System

ubuntu 22.04 LTS

**Comment by dbickson** [comment]
Thanks @zczczc11122 for your note. Python 3.8 is no longer supported. You need to upgrade to python3.9.
Thanks

**Comment by zczczc11122** [comment]
> 谢谢@zczczc11122感谢您的提醒。Python 3.8 不再受支持。您需要升级到 python3.9。 请参阅此弃用通知：[https](https://devcenter.heroku.com/changelog-items/2768) ://devcenter.heroku.com/changelog-items/2768 谢谢

Thanks for your reply.
But I tried to use python3.9 pip3 install fastdup==2.5, but I still got an error.

ERROR: Could not find a version that satisfies the requirement fastdup==2.5 (from versions: 1.85)
ERROR: No matching distribution found for fastdup==2.5

This seems to show that only 1.85 can be installed. If I now want to install version 2.5 through wheel, how should I obtain the wheel file of the corresponding version? I tried to find the wheel file corresponding to version 2.5 or 2.6 from the link https://github.com/visual-layer/fastdup/releases, but didn't find it either.
